debezium-ddl-parser/src/test/resources/mysql/examples/dml_select.sql

```diff
@@ -0,0 +1,128 @@
+#begin
+-- common conustructions tests
+-- -- Literals
+-- -- -- String literal
+SELECT 'hello world';
```

Comment:
Same question as above -- what's the motivation for all the DML tests?

Reply:
Actually i reused the SQL files for testing that was used by ANTLR grammar authors. According to select statements, they should be consistent because they are used for the creation of Views.